src/ResourceManager/Resources/Commands.ResourceManager/Cmdlets/Microsoft.Azure.Commands.ResourceManager.Cmdlets.dll-Help.xml

```diff
@@ -10236,5 +10236,259 @@ Remove-AzureRmPolicyDefinition -Id $policyDefinition.ResourceId -Force</dev:code
     <maml:relatedLinks>
     </maml:relatedLinks>
   </command:command>
+  <command:command xmlns:maml="http://schemas.microsoft.com/maml/2004/10" xmlns:command="http://schemas.microsoft.com/maml/dev/command/2004/10" xmlns:dev="http://schemas.microsoft.com/maml/dev/2004/10" xmlns:MSHelp="http://msdn.microsoft.com/mshelp">
+    <!--Generated by PS Cmdlet Help Editor-->
+    <command:details>
+      <command:name>Export-AzureRmResourceGroup</command:name>
+      <maml:description>
+        <maml:para>Captures the specifies resource group as a template and saves it to a file on disk.</maml:para>
+      </maml:description>
+      <maml:copyright>
+        <maml:para />
+      </maml:copyright>
+      <command:verb>Export</command:verb>
+      <command:noun>AzureRmResourceGroup</command:noun>
+      <dev:version />
+    </command:details>
+    <maml:description>
+      <maml:para>Captures the specifies resource group as a template and saves it to a file on disk.</maml:para>
```

Comment:
@ravbhatnagar Sure, can you please send me the exact text you want to be displayed here?